semgrep-core/src/matching/Generic_vs_generic.ml

```diff
@@ -427,7 +427,15 @@ let m_regexp_options a_opt b_opt =
 (* TODO: factorize with metavariable and aliasing logic in m_expr
  * TODO: remove MV.Id and use always MV.N?
  *)
-let rec m_name a b =
+(* `is_distributed` should be true if `b` is the result of unpacking a name which
```

Comment:
I liked `is_resolved` (or `is_imported`) better. Otherwise I would pick something more clear even it's longer, in the spirit of `has_tokens_from_many_places`. That one would be a totally fine name and much easier to understand than `is_distributed`.